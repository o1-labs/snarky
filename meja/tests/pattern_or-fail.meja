let (x | _) = 15;
