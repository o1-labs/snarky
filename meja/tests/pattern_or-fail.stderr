File "tests/pattern_or-fail.meja", line 1, characters 5-10:
Error: Variable x must occur on both sides of this '|' pattern.
